commit 17c3a1a1a22c468f3cfe4801e034fa76dbddb9aa (refs/changes/36/4536/3)
Author: Matthew Parsons <mparsons@mindless.com>
Date:   2018-07-28T16:19:11+00:00 (1 year, 2 months ago)
    
    joyent/smartos-live#690 trivial imgadm(1M) manpage fix (gz -> gzip)
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>
    Approved by: Joshua M. Clulow <jmc@joyent.com>

diff --git a/src/img/man/imgadm.1m.md b/src/img/man/imgadm.1m.md
index b56d6930..1d6d9e90 100644
--- a/src/img/man/imgadm.1m.md
+++ b/src/img/man/imgadm.1m.md
@@ -361,7 +361,7 @@ UUID.
                            then "PATH.imgmanifest" and "PATH.zfs[.EXT]" are
                            created.
             -c <comp>, --compression=<comp>
-                           One of "none", "gz", "bzip2" or "xz" for the compression
+                           One of "none", "gzip", "bzip2" or "xz" for the compression
                            to use on the image file, if any. Default is "none".
             -i             Build an incremental image (based on the "@final"
                            snapshot of the source image for the VM).
